### PR TITLE
Feature/2499/show garbage bin next to the loaded maps

### PR DESCRIPTION
**Pull request by Shivan-997**
# show garbage bin next to the loaded maps

closes: #2499 

## Screenshots or gifs
image

### CODE REVIEW

**Comment by Shivan-997** [comment]
i am not sure if it is the best solution to just disable button when only one map left.

File: visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/filePanelFileSelector.component.html [hunk 1]
```diff
@@ -15,6 +15,10 @@
 	<mat-option *ngFor="let file of files" [value]="file.file.fileMeta.fileName">
 		{{ file.file.fileMeta.fileName }}
 
-		<cc-remove-file-button [filename]="file.file.fileMeta.fileName" [onRemoveFile]="handleOnRemoveFile"></cc-remove-file-button>
+		<cc-remove-file-button
+			[filename]="file.file.fileMeta.fileName"
+			[onRemoveFile]="handleOnRemoveFile"
+			[files]="files"
```
Comment (cschild) [inline visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/filePanelFileSelector.component.html:21]: we could test the number of files here *ngIf="files.length > 1", thus the delete button wouldn't apear

File: visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/filePanelFileSelector.component.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+.mat-option-text {
+	display: flex !important;
```
Comment (Christian-Eberhard) [inline visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/filePanelFileSelector.component.scss:2]: !important is evil, try beeing more specific with the selector instead.

File: visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/removeFileButton/removeFileButton.component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<button mat-button (click)="onRemoveFile(filename, $event)" title="Remove map">
+	<span class="symbol"><i class="fa fa-trash delete-symbol"></i></span>
```
Comment (Christian-Eberhard) [inline visualization/app/codeCharta/ui/filePanel/filePanelFileSelector/removeFileButton/removeFileButton.component.html:2]: Is there a reason for the span? Maybe just give the <i> the class "symbol"

**Review by Christian-Eberhard — CHANGES_REQUESTED** [review]

**Review by Christian-Eberhard — APPROVED** [review]

**Pull request merged**